### ISSUE TITLE
verify newly generated coins are factored in under the approval colum of delegate monitor

### ISSUE THREAD

**Issue by Gr33nDrag0n69** [LiskHQ/lisk-explorer#11]
API return good values but in the explorer UI the approval percentage seem wrong. Maybe my logic is flawed, not sure.

![lisk-explorer-issue-greenshot](https://cloud.githubusercontent.com/assets/18356726/17534927/bb873f42-5e59-11e6-8af5-e31bcbab7b63.png)

joel is voting newcomer in on testnet with lisk balance from genesis block (98.99%)
So "in theory" everybody without this "genesis account" vote should be < 1.01%

There is about 90 of delegate with up to 1.66% of vote weight.
I think new generated coin are not factored in on the % of voting "approval"

**Comment by fix** [comment]
logic is flawed, genesis account has 97.99% of lisk :) https://testnet-explorer.lisk.io/address/6365926013346518016L

**Comment by Gr33nDrag0n69** [comment]
my logic or explorer internal logic ?

**Comment by Gr33nDrag0n69** [comment]
Reading some of the issue on the lisk repository, I saw this bug filled.
https://github.com/LiskHQ/lisk/issues/256
Maybe this is linked to this bug in the explorer.

**Comment by Isabello** [comment]
Root cause is here

https://github.com/LiskHQ/lisk-explorer/blob/18e5ecbebd180be65579bdc39cdceb9a80588e47/public/src/js/filters.js#L92

```
  .filter('approval', function () {
      return function (votes) {
          if (isNaN(votes)) {
              return 0;
          } else {
              return ((parseInt(votes) / 10000000000000000) * 100).toFixed(2);
          }
      };
  })
```

We are checking against static supply rather than the supply returned from `/api/blocks/getSupply`

**Comment by Isabello** [comment]
This function will still need work, or it might be ok to just remove entirely since we can just leverage `approval` response from each delegate rather than their vote total.


These two commits correct this issue

https://github.com/LiskHQ/lisk-explorer/commit/3e7e008bc18671724167d0e71f199337add02b2a

https://github.com/LiskHQ/lisk-explorer/commit/65c9158edede6c2a3123ff6cbcd15786f77a1f8c

**Comment by Isabello** [comment]
Addressed by #18
